### DISCUSSION TITLE
How to help the translation?

### DISCUSSION THREAD

**Discussion by mehr32** [elk-zone/elk#2491] [General]
Hello, I want to help with Farsi translation for the project. Can you guide me?

**Comment by Sma11X** [comment]
You can also refer to other closed PRs that create new translations.
1. add an object to `/config/i18n.ts` at `locales` for the language
2. create a json file at `/locales` then translate as others

**Comment by userquin** [comment]
Check the contributing guide:
https://github.com/elk-zone/elk/blob/main/CONTRIBUTING.md
